cover-checker-console/src/main/java/com/naver/nid/cover/util/ParameterParser.java

```diff
@@ -48,17 +50,30 @@ public Parameter getParam(String... commandArgs) {
 			return null;
 		}
 
+		final List<String> coveragePaths = Arrays.asList(cmd.getOptionValues("c"));
+		final String[] coverageTypesArray = cmd.getOptionValues(COVERAGE_TYPE_OPTION);
+		List<String> coverageTypes = Arrays.asList(
+				coverageTypesArray == null
+						? new String[]{ "jacoco" }
```

Comment:
Consider using an empty string here - that will make it closer to the original code. 

Otherwise, you have 2 places guessing what coverage parser to use. 
1. This place in `ParameterParser`
2. In `ObjectFactory`:
```
    public List<CoverageReportParser> getCoverageReportParser() {
        return IntStream.range(0, param.getCoveragePath().size())
                .mapToObj(index -> {
                    final String coverageType = param.getCoverageType().get(index);
                    if ("cobertura".equals(coverageType)) {
                        return new XmlCoverageReportParser(new CoberturaCoverageReportHandler());
                    } else {
                        return new JacocoReportParser();
                    }
                }).collect(Collectors.toList());
```
Alternatively, you could change the code in `ObjectFactory` to make it more strict.

Reply:
Good point. I've introduced a new `CoverageType` enum.